CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## UNRELEASED
 
+- Updated dependency for Phobos to 1.9.0-beta3. This ensures compatibility with
+  Phobos 2.0.
+  Fixed RSpec warning when using `test_consume_invalid_message`.
```

Comment:
Minor thing: Is this meant to be its own bullet or part of the Phobos change? (Missing `-`)

Reply:
I'll add the bullet :) Thanks!